### PR TITLE
Add nodemon and dotenv dependencies, update import statements, and us…

### PR DESCRIPTION
**Pull request by Agastya18**
added dotenv, also add  nodemon as  devDependencies  and use import statement by replacing existing code.

### CODE REVIEW

**Comment by Agastya18** [comment]
hey , could you check this PR

File: server/package-lock.json
Comment (Danny-Kyle) [inline server/package-lock.json:1]: lots of changes made on this one I see

File: server/app.js
Comment (Danny-Kyle) [inline server/app.js:1]: changing to ES6 then?

**Review by Danny-Kyle — COMMENTED** [review]
changes make sense and shoulld have no problem being accepted

File: server/app.js
Comment (Agastya18) [reply server/app.js:1]: yep

**Review by Danny-Kyle — APPROVED** [review]
Should have no problems, scaling to new ES6 module done efficiently

**Comment by KwameTech1** [comment]
Hi Danny,

I apologize for the delay in accepting the pull request. We recently moved the repository to a new account, which caused the delay.

Please check out the updated repository and submit a new pull request. Thank you for your understanding.

Best regards,
Edmond Akwasi.

**Pull request closed without merging**